userdocs/diagnostics/existential-any.md

```diff
@@ -1,24 +1,44 @@
-# `ExistentialAny`
+# Existential any (`ExistentialAny`)
 
-This diagnostic group includes errors and warnings pertaining to the `any` type
-syntax.
+`any` existential type syntax.
 
-This syntax was proposed in [SE-0335](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0335-existential-any.md).
-`any` syntax draws a line between constraint types and existential or boxed
-types.
+`any` was introduced in Swift 5.6 to explicitly mark "existential types", ie. boxed types that
```

Comment:
```suggestion
`any` was introduced in Swift 5.6 to explicitly mark "existential types", i.e., boxed types that
```

Reply:
I know that this is technically correct, but I do so hate it 😞. Seems like I snuck my `eg.` through though 😉